analyzers/tests/SonarAnalyzer.UnitTest/TestCases/PartialMethodNoImplementation.CSharp9.cs

```diff
@@ -1,18 +1,11 @@
 ﻿public partial record Record
 {
     partial void Method(); //Noncompliant {{Supply an implementation for this partial method.}}
```

Comment:
please add some tests with partial methods with non-void return and also with `out` parameters

Reply:
and please add an IT `S3251.cs` in the .NET 5 IT project